certification/internal/policy/operator/scc_check.go

```diff
@@ -0,0 +1,121 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/bundle"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+)
+
+// SecurityContextConstraintsCheck evaluates the csv and logs a message if a non default security context constraint is
+// needed by the operator
+
+type SecurityContextConstraintsCheck struct {
+	customSCCSpecified  bool
+	builtInSCCSpecified bool
+}
+
+func getDefaultSCCs() []string {
+	return []string{"restricted", "privileged", "nonroot", "hostnetwork", "hostmount-anyuid", "hostaccess", "anyuid"}
```

Comment:
Is there some other place we can get these from? Say the OpenShift API? As new SCCs come to be, I don't want us to have to remember these values.

Reply:
hmm we could do `oc get scc` but not sure if there is a way to filter built-in ones 🤔

Reply:
I wouldn't want to shell out to get the values, but I'd hope they are structs, constants in the API somewhere

Reply:
and too your point that list was _already_ out of date 😑 😅

Reply:
apparently not though : https://github.com/openshift/apiserver-library-go/blob/aa2bfcc66a43fc37769e3112642d7e089cce7913/pkg/securitycontextconstraints/sccadmission/admission.go#L140

Reply:
I created the below issue in their repo
- Relates: openshift/apiserver-library-go#88